tools/uart_hw_test/uart_test_windows.c

```diff
@@ -0,0 +1,312 @@
+/*******************************************************************
+*
+* Copyright 2022 Samsung Electronics All Rights Reserved.
+*
+* File Name : trap.c
+* Description: Receive RAM and/or Userfs dump using UART
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+******************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/time.h>
+#include <windows.h>
+
+/****************************************************************************
+ * Preprocessor definitions
+ ****************************************************************************/
+#define UART_PATH_STR		"\\\\.\\COM%s"
+#define UART_PATH_STR_BELOW_TEN		"COM%s"
```

Comment:
We could have a better naming of these macros instead of explicitly stating "BELOW_10"
Some way to unify the macros?